### PR TITLE
list signatories for group accounts

### PR DESCRIPTION
**Pull request by tahabi**
is useful when confirming group accounts

### CODE REVIEW

File: acct/check [hunk 1]
```diff
@@ -154,6 +154,13 @@ if [ -n "$User_Info" ]; then
   echo "-------------------------------------------------------------------------------"
 fi
 
+# list signatories for group account
+if [ -n "$callinkOid" ]; then
+    echo
+    echo "Signatories for group:";
+    echo "$(python3 -c "from ocflib.ucb.groups import signatories_for_group; print('\n'.join(map(lambda obj: obj['name'], signatories_for_group($callinkOid).values())));")"
```
Comment (chriskuehl) [inline acct/check]: can you just call `signat user <user>`?

Comment (tahabi) [inline acct/check]: img

File: acct/check [hunk 2]
```diff
@@ -155,11 +155,10 @@ if [ -n "$User_Info" ]; then
...
 # list signatories for group account
-if [ -n "$callinkOid" ]; then
-    echo
-    echo "Signatories for group:";
-    echo "$(python3 -c "from ocflib.ucb.groups import signatories_for_group; print('\n'.join(map(lambda obj: obj['name'], signatories_for_group($callinkOid).values())));")"
-fi
+echo 
+echo "Signatories/Signatory for:"
+signat user $user
+
```
Comment (kpengboy) [inline acct/check]: Trailing space :(

Comment (tahabi) [inline acct/check]: ?

**Pull request merged**